workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/ViewInitializer.kt

```diff
@@ -0,0 +1,28 @@
+package com.squareup.workflow1.ui
+
+import android.view.View
+
+/**
+ * TODO kdoc
+ */
+@WorkflowUiExperimentalApi
+public fun interface ViewInitializer {
+  /**
+   * TODO kdoc
+   *
+   * Implementations of this function _must_ call [showRendering].
+   */
+  public fun onViewCreated(view: View)
```

Comment:
Note to self: If it's important that `ViewStateCache` only call `restoreSavedInstanceState` _after_ the view's initial `showRendering`, then this will either have to be changed to allow initializers to run code both before and after the `showRendering` call, or we'll need two `update` functions that `BackStackContainer`s will be required to call. 

I'm leaning towards the latter – it keeps this API much simpler and more focused, and while it makes the `ViewStateCache` more complex, the extra verbosity might actually make the code more clear.

Reply:
initView is called in the same point in the initialization process, so they are related. 
But it's different than initView in that it's not tied to a particular view factory, but to a particular invocation of buildView.

doShowRendering is unrelated though, that gets called on every update.